aten/src/ATen/cpu/vec256/vec256_float_neon.h

```diff
@@ -319,11 +319,21 @@ template <> class Vec256<float> {
     __at_align32__ float tmp_exp[size()];
     store(tmp);
     exp.store(tmp_exp);
-    for (int64_t i = 0; i < size(); i++) {
+    for (int i = 0; i < size(); i++) {
```

Comment:
type `i` like `size()`:

```
auto i = decltype(size()){0}
```

Reply:
I really don't like `decltype(size())` so added a `using size_type` to `Vec256` that I can use instead.